cs/cli/vw_arguments.h

```diff
@@ -5,10 +5,13 @@
 #pragma once
 
 #include <msclr\marshal_cppstd.h>
-#include "vw.h"
-#include "config/cli_options_serializer.h"
+
 #include <algorithm>
 
+#include "config/cli_options_serializer.h"
```

Comment:
I've typically tried to include system headers after the local ones since it helps avoid order-dependency in include files. That way you'll get an error if a header is missing something

Reply:
Discussed directly. Will make this change in a separate PR.